### ISSUE TITLE
Documentation Query

### ISSUE THREAD

**Issue by Bcrutt** [springfall2008/batpred#583]
Hi
I noticed this section titled "None" in the documentation, screenshot below

![image](https://github.com/springfall2008/batpred/assets/132393335/4aa93eb6-a11f-4a52-a862-0c0156e509b5)

![image](https://github.com/springfall2008/batpred/assets/132393335/f2e84ffc-2877-464c-adf2-2a1f657c1bc5)

**Comment by gcoan** [comment]
That was my fault.

As part of the recent documentation changes in #570 7.14.31, I renamed config-yaml-settings.md to apps-yaml.md to reflect the correct file name.

Changed links from the install guide etc, but I missed editing the menu structure that referred to it.

Saw the problem myself this morning and included a fix in PR #581 (along with a few more documentation improvements).
It's waiting for @springfall2008 to review and approve 

Cheers Geoffrey
